client/my-sites/pages/page/index.js

```diff
@@ -808,9 +814,9 @@ const mapDispatch = {
 	restorePost,
 	setPreviewUrl,
 	setLayoutFocus,
+	recordTracksEvent,
 	recordEvent,
 	recordMoreOptions: () => recordEvent( 'Clicked More Options Menu' ),
-	recordPageTitle: () => recordEvent( 'Clicked Page Title' ),
```

Comment:
I would not remove this event so we can still call it from `recordPageTitleClick`

Reply:
I see